mergestat/mergestat
# fix(repos): Show default page instead of the empty page

Closes #536 
To test this you need to :

1. Log in to mergestat and validate you credentials
2. Add a single repo to your repo page
3. Remove that repo
4. You should see the next picture
<img width="1440" alt="Screen Shot 2023-01-11 at 7 26 06 PM" src="https://user-images.githubusercontent.com/43893061/211947083-b2b60a07-bc87-48e3-a27f-9ff9adc97bde.png">
